pkg/proctree/events_processing.go

```diff
@@ -0,0 +1,39 @@
+package proctree
+
+import (
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+// ProcessEvent update the process tree according to arriving event
+func (tree *ProcessTree) ProcessEvent(traceeEvent *trace.Event) error {
+	switch traceeEvent.EventName {
+	case "sched_process_fork":
+		return tree.processForkEvent(traceeEvent)
+	case "sched_process_exec":
+		return tree.processExecEvent(traceeEvent)
+	case "sched_process_exit":
+		return tree.processExitEvent(traceeEvent)
+	case "exit", "init_namespaces":
```

Comment:
Add to excluded list all the user-mode events.
Even better - think of a way to identify non-kernel events and prevent their processing.